packages/mock/src/rekor/tlog.ts

```diff
@@ -104,25 +139,12 @@ class TLogImpl implements TLog {
     // Assemble checkpoint from note and signature
     const checkpoint = [note, sigLine, ''].join('\n');
 
-    const proof: InclusionProof = {
-      logIndex: logIndex,
+    return {
+      logIndex: 0,
       treeSize: treeSize,
       checkpoint,
-      hashes: [leafHash.toString('hex')],
+      hashes: [],
```

Comment:
Do you not need any hashes here?

Reply:
@feelepxyz as a result of fixing that verification bug we can now represent the simplest possible merkle tree which has a single node. In this scenario, the root hash of the tree is equal to the leaf hash of the sole node -- no other node hashes are required to verify the integrity of the tree.